packages/fadbadml/fadbadml.0.1/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "FADBAD++ for OCaml"
+maintainer: [
+  "francois-bidet <fbidet@lix.polytechnique.fr>"
+  "ismailbennani <ismail.lahkim.bennani@ens.fr>"
+]
+authors: [
+  "francois-bidet <fbidet@lix.polytechnique.fr>"
+  "ismailbennani <ismail.lahkim.bennani@ens.fr>"
+]
+license: "CeCILL-C"
+homepage: "https://github.com/ismailbennani/FADBADml"
+bug-reports: "https://github.com/ismailbennani/FADBADml/issues"
+depends: [ "ocaml" { >= "4.08" } "ocamlfind" ]
```

Comment:
```suggestion
depends: [
  "ocaml" {>= "4.08"}
  "ocamlfind" {build}
]
```
ocamlfind should be tagged as a build dependency

Reply:
thanks !